testing/src/main/java/org/oppia/android/testing/uiautomator/EndToEndTestHelper.kt

```diff
@@ -0,0 +1,63 @@
+package org.oppia.android.testing.uiautomator
+
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiObject2
+import androidx.test.uiautomator.UiScrollable
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until
+import com.google.common.truth.Truth.assertThat
+
+/** This object contains common operations used for end-to-end tests. */
+object EndToEndTestHelper {
+
+  private val OPPIA_PACKAGE = "org.oppia.android"
+  private val LAUNCH_TIMEOUT = 30000L
+  private val TRANSITION_TIMEOUT = 5000L
```

Comment:
These names should include their corresponding units.

Reply:
Does that mean something like 
`LAUNCH_TIMEOUT_30_SECONDS`
`TRANSITION_TIMEOUT_5_SECONDS` ??

Reply:
Not quite. The number shouldn't be included since that's an implementation detail. Instead: ``LAUNCH_TIMEOUT_SECONDS`` and ``TRANSITION_TIMEOUT_SECONDS`` are more of what I was looking for.